src/components/Organisms/EventsHeader/constants.jsx

```diff
@@ -29,4 +30,9 @@ export const navButtons = [
     label: 'video',
     href: 'https://www.youtube.com/channel/UCqwHhJNEUe7D-HGsX4zvKzQ',
   },
+  {
+    icon: <BookIcon />,
+    label: 'trainings',
```

Comment:
It is possible to translate this as well? `Trainings` is a bit weird in Dutch;

![screen shot 2018-11-16 at 07 51 27](https://user-images.githubusercontent.com/21951794/48603303-4b16d380-e976-11e8-8482-47ce16c6faf3.png)

Reply:
This was of course way more work: it did not support translations yet in this place. And then I wanted to add typings. And than some typing was missing, so I had to add one. And then I thought I might as well add that new typing all over the place...